### ISSUE TITLE
Feedbacks for Note11

### ISSUE THREAD

**Issue by SerenaQYHuang** [hupili/python-for-data-and-media-communication-gitbook#109]
1. The output for this piece of code is not the same with the DEFAULT
![image](https://user-images.githubusercontent.com/41979788/48975189-19081e80-f0a6-11e8-8cd6-89ac60dda723.png)
迷茫

**Comment by SerenaQYHuang** [comment]
2. A question concerning time delta
```
from datetime import datetime
datetime(2018, 6, 12, 0, 0) - datetime(2018, 2, 3, 0, 0)
```
What if I'd like to know how many days there are from now to a particular time?
I tried the following:
```
datetime(2018, 12, 4, 17, 0) - datetime.now()
```
but got an error like this
![image](https://user-images.githubusercontent.com/41979788/48975610-8e2c2180-f0af-11e8-9996-78a115cfd58f.png)

**Comment by SerenaQYHuang** [comment]
![image](https://user-images.githubusercontent.com/41979788/48975771-4d360c00-f0b3-11e8-97f6-ed8dc799a0bb.png)

It turns out we are unable to get access to this csv file, making it difficult to run the whole time series analysis practice @hupili @ChicoXYC

**Comment by SerenaQYHuang** [comment]
Note 12:
First, I tried to draw the graph using nx.show(g)
Nothing happened
![image](https://user-images.githubusercontent.com/41979788/48976741-3c8f9100-f0c7-11e8-8a76-268cd255d1da.png)

Then, I added plt.show(), and I got this
![image](https://user-images.githubusercontent.com/41979788/48976747-634dc780-f0c7-11e8-9fbe-a762b079a1d9.png)

This is weird because there are only three nodes in my graph
I tried to restart and run again, but the result was the same @hupili @ChicoXYC

**Comment by FLYSTEPHEN** [comment]
You are right. If we use `print()`, the output will drop its type traits to get a better effect of display on the screen. Just use `dt_1,dt_2,dt_3,dt_4` without `print()` in the last line to get the same result in this note. I will revise this point.

**Comment by hupili** [comment]
@SerenaQYHuang @ChicoXYC , please redirect the week 12 to another thread. It is better the keep the topic focused in one issue.

**Comment by hupili** [comment]
@FLYSTEPHEN , I think the OP may be asking the function of "default" --- i.e. when there is any datetime dimentions (year/ month/ ...) missing, the parser will use default in that place. We can elaborate this in the notes.

@SerenaQYHuang , can you further elaborate the question? Which part is confusing? You can also check if [this PR](https://github.com/hupili/python-for-data-and-media-communication-gitbook/pull/110/files) solves your problem.

**Comment by SerenaQYHuang** [comment]
@hupili The problem is solved. I was confused by the format of the result because it did not match the DEFAULT but @FLYSTEPHEN has pointed out that it was because I used the function ```print()```. I tried and the problem was solved

**Comment by FLYSTEPHEN** [comment]
@hupili 's comment is also right. We need to explain the function of `default` here. I will add  it later

**Comment by ConnorLi96** [comment]
## Other feedbacks in Chapter 11

- **In the A failed parsing case**
There are Chinese characters in ```parse("2月15日 10:36:28")```, and then it failed. I didn't find solution in references, I guess add it in notebook will be better.

- **In the Add timedelta to a datetime object**
The sample code seem has error, and I run below code successfully.
<img width="852" alt="image" src="https://user-images.githubusercontent.com/41864925/49006379-efcdb800-f1a3-11e8-8ad9-f5122ee80e5c.png">

- **In the Resample, aggregate and plot**
I try to read csv by link in [this notebook](https://github.com/hupili/python-for-data-and-media-communication/blob/master/datetime/timeseries.ipynb), but there will be error. Although I copied and pasted the link in Chrome, there was 404 error.

**Comment by ChicoXYC** [comment]
@ConnorLi96 the link in the notebook you give is outdated, the csv in the notes is the newest. But the strange thing is that I still cannot open this csv with the following error:

```text
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)>
```

The alternative solution:

```python
import pandas as pd
import io
import requests
url="https://raw.githubusercontent.com/hupili/python-for-data-and-media-communication/master/text-analysis/regular_reader_tweets.csv"
s=requests.get(url).text
df=pd.read_csv(io.StringIO(s))
```

@hupili  do you know this, sir?

**Comment by hupili** [comment]
talked about this during class. It is likely to be CVA problem. That is why in the class demo of week07, I download the data from GitHub, instead of directly loading via URL

**Comment by hupili** [comment]
@FLYSTEPHEN , file location can refer to [this comment](https://github.com/hupili/python-for-data-and-media-communication-gitbook/issues/109#issuecomment-441618735)
